typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -974,6 +974,31 @@ prefab types with the (implicitly quoted) prefab-key
 
 @history[#:added "1.7"]}
 
+@defform[(Struct-Property ty)]{
+  Internal type that describes a property value that is attached to a
+   structure type.
```

Comment:
This documentation should specify what `ty` means; that values associated with the property must have type `ty`. Would an example like this help?
```racket
;; prop : (Struct-Property ty)
(struct s [] #:property prop expr)
;; requires expr : ty
```